app.json

```diff
@@ -3,8 +3,8 @@
     "name": "campo-libre",
     "description": "This project is really great.",
     "slug": "campo-libre",
-    "privacy": "public",
-    "sdkVersion": "25.0.0",
+    "privacy": "unlisted",
```

Comment:
Make the app private by default on Expo's site. We'll make it public when we're ready to show it.